### PR TITLE
#127 - check if there exist meaningful hyperledger log entry variables

### LINKED ISSUE #127: Check if there exist meaningful Hyperledger LogEntryVariables and add them to BLF

### PR DESCRIPTION
**Pull request by PaulsBecks**
Closes #127.

No there do not exist any meaningful variables. All variables that are useful are already extracted from the parameters given to the logEntryFilter itself.

### CODE REVIEW

File: src/main/antlr4/blf/grammar/BcqlLexemes.tokens [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+'indexed'=20
+'!'=21
+'&&'=22
+'||'=23
+'_indexed_'=26
+'_'=27
+'address'=34
+'bool'=35
+'u'=37
+'string'=40
```
Comment (mykola-digtiar) [inline src/main/antlr4/blf/grammar/BcqlLexemes.tokens]: Why do we need this file?

Comment (PaulsBecks) [reply src/main/antlr4/blf/grammar/BcqlLexemes.tokens]: We don't. I will remove it.

**Review by Knostpe — DISMISSED** [review]

**Review by daniil-romanchenko — DISMISSED** [review]
Ok, tests are running fine, unnecessary files are removed. 
Seems fine!

**Comment by mykola-digtiar** [comment]
Fixed last commit message.

**Review by mykola-digtiar — APPROVED** [review]
👍

**Review by rohrschacht — APPROVED** [review]
Alright! :+1:

**Pull request merged**